variables.tf

```
@@ -39,6 +39,12 @@ variable "lacework_image_pull_policy" {
   default     = "Always"
 }
 
+variable "lacework_server_url" {
```

all variables are prepend with `lacework_agent_*`
```suggestion
variable "lacework_agent_server_url" {
```